packages/google_mlkit_document_scanner/ios/google_mlkit_document_scanner.podspec

```diff
@@ -0,0 +1,27 @@
+require 'yaml'
+
+pubspec = YAML.load_file(File.join('..', 'pubspec.yaml'))
+library_version = pubspec['version'].gsub('+', '-')
+
+Pod::Spec.new do |s|
+  s.name             = pubspec['name']
+  s.version          = library_version
+  s.summary          = pubspec['description']
+  s.description      = pubspec['description']
+  s.homepage         = pubspec['homepage']
+  s.license          = { :file => '../LICENSE' }
+  s.author           = 'Multiple Authors'
+  s.source           = { :path => '.' }
+  s.source_files = 'Classes/**/*'
+  s.public_header_files = 'Classes/**/*.h'
+  s.dependency 'Flutter'
+  #   s.dependency 'GoogleMLKit/DocumentScanner', '~> 4.0.0'
+  s.dependency 'google_mlkit_commons'
```

Comment:
remove this line, this package does not have this dependency

Reply:
removed. 
under this PR 
https://github.com/flutter-ml/google_ml_kit_flutter/pull/609